Modules/itertoolsmodule.c

```diff
@@ -4590,11 +4590,11 @@ chain(p, q, ...) --> p0, p1, ... plast, q0, q1, ... \n\
 chain.from_iterable([p, q, ...]) --> p0, p1, ... plast, q0, q1, ... \n\
 compress(data, selectors) --> (d[0] if s[0]), (d[1] if s[1]), ...\n\
 dropwhile(pred, seq) --> seq[n], seq[n+1], starting when pred fails\n\
-groupby(iterable[, keyfunc]) --> sub-iterators grouped by value of keyfunc(v)\n\
+groupby(iterable, key=None) --> sub-iterators grouped by value of keyfunc(v)\n\
```

Comment:
Changed in b6336f5. 

I'm a bit worried it looks confusing to someone that hasn't used group by before. Someone might assume that it defaults to None, and will assume that not providing a function will cause a crash when a None value is called.